opennms-container/launch_apt_server.sh

```diff
@@ -83,12 +83,11 @@ while [ "$COUNT" -lt 120 ]; do
   sleep 1
 done
 
+echo "docker logs:"
 docker logs "${APT_CONTAINER_NAME}"
 
-if [ "$COUNT" -eq 30 ]; then
-  echo "gave up waiting for server"
-  echo "docker logs:"
+if [ "$COUNT" -eq 120 ]; then
```

Comment:
A minor suggestion. Use a variable to store 120 and share between Line:77. So no one will miss the number again.